examples/blueprint-construct/index.ts

```diff
@@ -326,52 +327,20 @@ function addCustomNodeGroup(): blueprints.ManagedNodeGroup {
 }
 
 function addWindowsNodeGroup(): blueprints.ManagedNodeGroup {
-    
-    const windowsUserData = ec2.UserData.forWindows();
-    windowsUserData.addCommands(`
-      $ErrorActionPreference = 'Stop'
-      $EKSBootstrapScriptPath = "C:\\\\Program Files\\\\Amazon\\\\EKS\\\\Start-EKSBootstrap.ps1"
-      Try {
-        & $EKSBootstrapScriptPath -EKSClusterName 'blueprint-construct-dev'
-      } Catch {
-        Throw $_
-      }
-    `);
-    const ebsDeviceProps: ec2.EbsDeviceProps = {
-        deleteOnTermination: false,
-        volumeType: ec2.EbsDeviceVolumeType.GP2
-    };
 
     return {
```

Comment:
What prompted these changes? Are they intentional?

Reply:
WIndows pattern prompted these changes. So we can use Disksize which takes care of EBS volume so we dont need to use LT.